lib/okapi/endpoint-service.js

```diff
@@ -0,0 +1,37 @@
+const OkapiClient = require('./okapi-client');
+
+const parseJson = data => data.json();
+const stringifyJson = data => JSON.stringify(data, null, '  ');
+
+module.exports = class EndpointService {
+  constructor(okapi, tenant) {
+    this.client = new OkapiClient(okapi, tenant);
+  }
+
+  get(path) {
+    return this.client
+      .get(path)
+      .then(parseJson)
+      .then(stringifyJson);
+  }
+
+  post(path, body) {
+    return this.client
+      .post(path, body)
+      .then(parseJson)
+      .then(stringifyJson);
+  }
+
+  put(path, body) {
+    return this.client
+      .put(path, body)
+      .then(parseJson)
+      .then(stringifyJson);
+  }
+
+  delete(path) {
+    return this.client
+      .delete(path)
+      .then(response => response.status >= 200 && response.status < 300);
```

Comment:
Ah-ha! Some logic for this class... but wait, the okapi-client already [takes care of this](https://github.com/folio-org/stripes-cli/blob/6226b8f5703facd4e9301a98167e7e0b1316f4a1/lib/okapi/okapi-client.js#L7-L16) for you.